library/css/components/nested-vertical-list.css

```diff
@@ -0,0 +1,80 @@
+.vertical-list-nest {
+    display: none;
+}
+
+.vertical-list-head {
+    cursor: pointer;
+    margin-bottom: 21px;
+}
+
+.faculty-vertical-list .first {
+    width: 15vw !important;
+}
+
+.faculty-vertical-list .second {
+    width: 60vw !important;
+    overflow-y: auto;
+    margin-top: 5rem !important;
+    padding-top: 0 !important;
+    position: relative;
+}
+
+.vertical-list-nest > div:hover {
+    color: #0067B3DE;
+}
+
+.faculty-vertical-list {
+    border: 1px solid #EEEEEE;
+    filter: drop-shadow(0px 4px 12px rgba(134, 134, 134, 0.1));
+    border-radius: 4px;
+    padding-bottom: 4rem;
+}
+
+
+.faculty-vertical-list  .publication-list {
+    margin-bottom:3rem;
+}
+
+.vertical-list > hr {
+    color: rgba(81, 131, 184, 0.8);
+    margin-bottom: 21px;
+    margin-top: 21px;    
+}
+
+.vertical-list-nest > div {
+    cursor: pointer;
+    margin-bottom: 21px;
+}
+
+.active-list-item {
+    color: #0067B3DE !important;
+}
+
+.vertical-list {
```

Comment:
More selective css selector, same name can be used by someone else also in other components